app/lib/account/default_auth_provider.dart

```diff
@@ -249,26 +249,29 @@ class DefaultAuthProvider extends AuthProvider {
   @override
   Future<AccountProfile?> getAccountProfile(String? accessToken) async {
     if (accessToken == null) return null;
-    final client = httpRetryClient(innerClient: http.Client());
-    final authClient = auth.authenticatedClient(
-        client,
-        auth.AccessCredentials(
-          auth.AccessToken(
-            'Bearer',
-            accessToken,
-            clock.now().toUtc().add(Duration(minutes: 20)), // avoid refresh
-          ),
-          null,
-          [],
-        ));
+    final client = httpRetryClient(retries: 2);
+    try {
+      final authClient = auth.authenticatedClient(
+          client,
+          auth.AccessCredentials(
+            auth.AccessToken(
+              'Bearer',
+              accessToken,
+              clock.now().toUtc().add(Duration(minutes: 20)), // avoid refresh
+            ),
+            null,
+            [],
+          ));
 
-    final oauth2 = oauth2_v2.Oauth2Api(authClient);
-    final info = await oauth2.userinfo.get();
-    client.close();
-    return AccountProfile(
-      name: info.name ?? info.givenName,
-      imageUrl: info.picture,
-    );
+      final oauth2 = oauth2_v2.Oauth2Api(authClient);
+      final info = await oauth2.userinfo.get();
+      return AccountProfile(
+        name: info.name ?? info.givenName,
+        imageUrl: info.picture,
+      );
+    } finally {
+      client.close();
```

Comment:
I think we had an innerClient to ensure reuse of connections. Why do you not want that no

Reply:
We did recreate `client` here, I haven't changed that part, only made the closing of it explicit. On the other hand, we could just reuse `_httpClient` if we are happy to have the same retry counts.